api/client/service/update.go

```diff
@@ -247,6 +250,26 @@ func updatePlacement(flags *pflag.FlagSet, placement *swarm.Placement) {
 	placement.Constraints = removeItems(placement.Constraints, toRemove, itemKey)
 }
 
+func updateContainerLabels(flags *pflag.FlagSet, field *map[string]string) {
```

Comment:
Can we parameterize the `updateLabels()` function and re-use it? They should be the same types.